distribution/src/conf/rule-engine-config.xml

```diff
@@ -0,0 +1,12 @@
+<ruleEngineConfig xmlns="http://wso2.org/carbon/rules/conf">
```

Comment:
Is there any specific reason for adding this file here. It will be available with the feature itself https://github.com/wso2/carbon-rules/blob/master/features/mediation/org.wso2.carbon.rule.mediation.server.feature/resources/conf/rule-engine-config.xml. Can't we just pack that.

Reply:
Modified the bin.xml file to pack the config file which is available with the feature.
Thanks for pointing out.